binderhub/tests/test_build.py

```diff
@@ -17,17 +17,26 @@
 
 @pytest.mark.asyncio(timeout=900)
 @pytest.mark.parametrize("slug", [
-    "gh/binderhub-ci-repos/requirements/d687a7f9e6946ab01ef2baa7bd6d5b73c6e904fd",
-    "git/{}/d687a7f9e6946ab01ef2baa7bd6d5b73c6e904fd".format(
+    "gh/binderhub-ci-repos/requirements/20c4fe55a9b2c5011d228545e821b1c7b1723652",
+    "git/{}/20c4fe55a9b2c5011d228545e821b1c7b1723652".format(
         quote("https://github.com/binderhub-ci-repos/requirements", safe='')
     ),
     "git/{}/master".format(
         quote("https://github.com/binderhub-ci-repos/requirements", safe='')
     ),
-    "gl/minrk%2Fbinderhub-ci/0d4a217d40660efaa58761d8c6084e7cf5453cca",
```

Comment:
Please don't remove this test. Instead lets try and simplify the repository so it builds more quickly

Reply:
I agree that we should keep this test.